src/Tribe/Customizer.php

```diff
@@ -453,7 +455,19 @@ public function inline_style( $force = false ) {
 				 */
 				do_action( 'tribe_customizer_before_inline_style', $sheet, $inline_style );
 
-				wp_add_inline_style( $sheet, $inline_style );
+				// kldjfljskdf
```

Comment:
Best description.

Reply:
I disagree, I thought this one was much more descriptive https://github.com/moderntribe/tribe-common/pull/1509/files#diff-a52472ee99d88d2e9c04a5902ba0d3159d10ef3ac91afa1e35ae9dd2cc435b80R120